### PR TITLE
Add a script to install all required pip packages

### PR DESCRIPTION
**Pull request by zachmullen**
This also fixes the aiur dashboard

### CODE REVIEW

File: scripts/InstallPythonRequirements [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            print('\033[35m*** Skipping plugin: %s\033[0m' % path)
+            continue
+
+        pluginPath = os.path.join(pluginsDir, path)
+        if os.path.isdir(pluginPath):
+            installFromDir(pluginPath, isDevMode, args.pip)
+
+
+if __name__ == '__main__':
+    import argparse
```
Comment (cpatrick) [inline scripts/InstallPythonRequirements]: I favor imports at the top.

**Comment by cpatrick** [comment]
Could we make the python script end in .py?

**Comment by cpatrick** [comment]
other than the nits above, this lgtm.

**Comment by zachmullen** [comment]
@cpatrick

**Pull request merged**